packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts

```diff
@@ -149,4 +149,35 @@ describe('scaffolding component testing', {
       verifyConfigFile(`cypress.config.js`)
     })
   })
+
+  context('3rd party ct plugin', () => {
+    it('Scaffolds component testing for Qwik using Vite', () => {
+      cy.scaffoldProject('qwik-app')
+      cy.openProject('qwik-app')
+
+      cy.withCtx(async (ctx) => {
+        await ctx.actions.file.removeFileInProject('./node_modules/cypress-ct-qwik')
+        await ctx.actions.file.moveFileInProject('./cypress-ct-qwik', './node_modules/cypress-ct-qwik')
+      })
+
+      cy.visitLaunchpad()
+      cy.skipWelcome()
+
+      cy.contains('Component Testing').click()
+      cy.contains('button', 'Qwik').should('be.visible')
```

Comment:
There is a race condition causing CI to fail here. Introduced here, since getting frameworks is now an async operation, since we glob in `node_modules`.

We need to fix this before merging.